sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/filter/AadAuthenticationFilter.java

```diff
@@ -48,7 +48,7 @@
     private static final String CURRENT_USER_PRINCIPAL = "CURRENT_USER_PRINCIPAL";
 
     private final UserPrincipalManager userPrincipalManager;
-    private final AzureADGraphClient azureADGraphClient;
+    private final AzureAdGraphClient azureAdGraphClient;
```

Comment:
How about naming it `AadGraphClient` to follow the same styles as other classes?